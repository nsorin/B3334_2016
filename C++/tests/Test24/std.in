R r1 5 10 10 5
R r2 7 20 20 7
OI r1int2 r1 r2
MOVE r1int2 10 10
MOVE r1int2 10 10
MOVE r1int2 10 10
UNDO
UNDO
UNDO
HIT r1int2 8 9
REDO
REDO
REDO
HIT r1int2 8 9
HIT r1int2 8 6
HIT r1int2 8 12
EXIT
