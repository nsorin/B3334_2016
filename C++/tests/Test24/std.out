#R r1 5 5 10 10
OK
#R r2 7 7 20 20
OK
#OI r1int2 r1 r2
OK
#MOVE r1int2 10 10
OK
#MOVE r1int2 10 10
OK
#MOVE r1int2 10 10
OK
#UNDO
OK
#UNDO
OK
#UNDO
OK
#HIT r1int2 8 9
YES
#REDO
OK
#REDO
OK
#REDO
OK
#HIT r1int2 8 9
NO
#HIT r1int2 8 6
NO
#HIT r1int2 8 12
NO
#EXIT
